docs/source/deploy.rst

```diff
@@ -0,0 +1,127 @@
+.. _usage:
+
+Deploy ``repo2docker`` 
+======================
+
+We've created for you the `continuous-build <https://www.github.com/binder-examples/continuous-build/>`_ 
+repository so that you can deploy a `Docker <https://docs.docker.com/>`_ container 
+to `Docker Hub <https://hub.docker.com/>`_ directly from a Github repository
+that has a Jupyter notebook. Here are instructions to do this.
+
+Getting Started
+---------------
+Today you will be doing the following:
+
+ 1. Fork and clone the continuous-build Github repository to obtain the hidden ``.circleci`` folder.
+ 2. creating an image repository on Docker Hub
+ 3. connecting your repository to CircleCI
+ 4. push, commit, or create a pull request to trigger a build.
+
+You don't need to install any dependencies on your host to build the container, it will be done
+on a continuous integration server, and the container built and available to you
+to pull from Docker Hub.
+
+
+Step 1. Clone the Repository
+............................
+First, fork the `continuous-build <https://www.github.com/binder-examples/continuous-build/>`_ Github
+repository to your account, and clone the branch. 
+
+   git clone https://www.github.com/<username>/continuous-build
+   # or
+   git clone git@github.com:<username>/continuous-build.git
+
+
+Step 2. Choose your Configuration
+.................................
+
+The hidden folder `.circleci/config.yml` has instructions for `CircleCI <https://circleci.com/dashboard/>`_ 
+to automatically discover and build your repo2docker jupyter notebook container.
+The default template provided in the repository in this folder will do the most basic steps, 
+including:
+
+1. clone of the repository with the notebook that you specify
+2. build
+3. push to Docker Hub
+
+Also provided (or will be provided as they are requested) are other configuration templates in
```

Comment:
I'd change it to say what it does provide and avoid speculating about the future.

Reply:
```python
from __future__ import vanessa
```